src/main/java/com/lambdatest/jenkins/freestyle/api/Constant.java

```diff
@@ -20,22 +21,41 @@ public interface Constant {
 	String LT_BUILD_NUMBER = "LT_BUILD_NUMBER";
 	String LT_USERNAME = "LT_USERNAME";
 	String LT_ACCESS_KEY = "LT_ACCESS_KEY";
+	String LT_PLATFORM_NAME = "LT_PLATFORM_NAME";
+	String LT_BRANDS = "LT_BRANDS";
+	String LT_DEVICES = "LT_DEVICES";
+	String LT_BRAND_NAME = "LT_BRAND_NAME";
+	String LT_DEVICE_NAME = "LT_DEVICE_NAME";
+	String LT_DEVICE_VERSION = "LT_DEVICE_VERSION";
+	String LT_APP_URL = "LT_APP_URL";
 
 	String OPERATING_SYSTEM = "operatingSystem";
 	String BROWSER_NAME = "browserName";
 	String BROWSER_VERSION = "browserVersion";
 	String RESOLUTION = "resolution";
 
+	String PLATFORM_NAME = "platformName";
+	String BRAND_NAME = "brandName";
+	String DEVICE_NAME = "deviceName";
+	String DEVICE_VERSION = "deviceVersion";
+	String APP_ID = "appUrl"; 
```

Comment:
Can we change it to `APP_URL` as per the value

Reply:
There is already a variable called APP_URL. So, can I change the variable name or value inside the variable?

Reply:
Done